tests/boards/nrf/qdec/boards/nrf54l15pdk_nrf54l15_common.dtsi

```diff
@@ -28,6 +28,14 @@
 				<NRF_PSEL(QDEC_B, 1, 10)>;
 		};
 	};
+
+	qdec_sleep_pinctrl: qdec_sleep_pinctrl {
+		group1 {
+			psels = <NRF_PSEL(QDEC_A, 1, 8)>,
+				<NRF_PSEL(QDEC_B, 1, 10)>;
+			low-power-enable;
+		};
+	};
```

Comment:
shouldn't this be part of board? otherwise, in any other app suspend will fail.

Reply:
not sure. we don't know on which pins QDEC is used.